testsuite/single_node_performance.py

```diff
@@ -90,6 +91,10 @@ class RunGroupConfig:
 # https://app.axiom.co/aptoslabs-hghf/explorer?qid=29zYzeVi7FX-s4ukl5&relative=1
 # fmt: off
 TESTS = [
+    RunGroupConfig(expected_tps=10000, key=RunGroupKey("econia-basic1-market"), expected_stages=10, included_in=Flow.ECONIA),
+    RunGroupConfig(expected_tps=10000, key=RunGroupKey("econia-advanced1-market"), expected_stages=10, included_in=Flow.ECONIA),
+    RunGroupConfig(expected_tps=10000, key=RunGroupKey("econia-advanced20-market"), expected_stages=10, included_in=Flow.ECONIA),
```

Comment:
I've implemented code for setting any number of markets with `num_markets` parameter. As it gave an error during execution, I realized we can't register multiple markets with the same trading pair in an account. So right now, we can have multiple markets by setting NUM_MODULES accordingly.